pkgs/development/libraries/subunit/default.nix

```
@@ -1,17 +1,19 @@
-{ stdenv, fetchurl, pkgconfig, check, cppunit, perl, pythonPackages }:
+{ stdenv, fetchFromGitHub, pkg-config, check, cppunit, perl, pythonPackages }:
 
 # NOTE: for subunit python library see pkgs/top-level/python-packages.nix
 
 stdenv.mkDerivation rec {
   pname = "subunit";
-  version = "1.1.0";
+  version = "1.3.0";
 
-  src = fetchurl {
-    url = "https://launchpad.net/subunit/trunk/${version}/+download/${pname}-${version}.tar.gz";
-    sha256 = "0lcah7p66c05p7xpw6ns1is0i02lh0nq8gq51mv4wyvbr6zaasa8";
+  src = fetchFromGitHub {
+    owner = "testing-cabal";
+    repo = pname;
+    rev = version;
+    sha256 = "1hk5v8gg5azwrc079xqigj0sp6qm47vzg1qwjbvb8dbjwlg75rvz";
   };
 
-  nativeBuildInputs = [ pkgconfig ];
+  nativeBuildInputs = [ pkg-config ];
```

enable creation of Makefile
```suggestion
  nativeBuildInputs = [ autoreconfHook pkg-config ];
```